ls1intum/Artemis
# [Exam] Bugfix: fix issue with dnd layout for drag items

### Checklist

### Motivation and Context


Fixes: https://github.com/ls1intum/Artemis/issues/1910
Fixes #1795 

### Description

Fixes issues with the displaying of drag and drop questions
- Added max length for the container of drag items for participation
- Removed onResize() (see comment of @schultek )

### Steps for Testing


1. Log in to Artemis
2. Create Exam with drag and drop questions
3. Participate on the exam
4. Make sure that no strange behaviour appears during participation of dnd questions
5. Make sure that in the summary the dnd questions are displayed correctly
6. Make sure that when clicking on print the dnd questions are shown correctly

### Screenshots


<img width="832" alt="image" src="https://user-images.githubusercontent.com/41108487/87862094-9a33b980-c94c-11ea-862c-78ccc6d9a249.png">
<img width="1170" alt="image" src="https://user-images.githubusercontent.com/41108487/87862120-ebdc4400-c94c-11ea-94d7-773ed17754f0.png">
<img width="1116" alt="image" src="https://user-images.githubusercontent.com/41108487/87862124-f26abb80-c94c-11ea-9de5-1b48fc536c14.png">

Print: 
[Exam.pdf](https://github.com/ls1intum/Artemis/files/4942578/Exam.pdf)
